src/test/kotlin/car/domain/CarTest.kt

```diff
@@ -0,0 +1,39 @@
+package car.domain
+
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class CarTest {
+    @ParameterizedTest
+    @ValueSource(ints = [0, 1, 2, 3])
+    fun `랜덤이 4 미만이고 움직일 경우 포지션은 0이여야 한다`(rand: Int) {
+        // given
+        val car: Car = Car(CarName("오길환"))
```

Comment:
생성자의 수가 늘어날수록 클래스를 사용하기 위한 유연성은 높아집니다.

```suggestion
        val car: Car = Car("오길환")
```

Reply:
이 부분에 고민을 조금 했는데, String 인자를 받는게 더 유연하겠네요!! 감사합니다